INSTALL.md

```diff
@@ -11,21 +11,59 @@ Before installing Goldstone, your server must meet the following prerequisites:
 
 To view and use Goldstone, you'll need a recent version of [Firefox](https://www.mozilla.org/en-US/firefox/products/), [Safari](https://www.apple.com/safari/), or [Chrome](https://www.google.com/intl/en-US/chrome/browser).
 
-## Install RPM and run the configuration script (as root)
+## Install RPMs (as root)
 
-Download the [lastest release](https://github.com/Solinea/goldstone-server/releases) and execute with the following steps:
+Download the [latest release](https://github.com/Solinea/goldstone-server/releases) and execute these commands:
 
 ```bash
 root# yum update ; reboot
 root# yum localinstall -y goldstone-server-{version}.rpm
+```
+
+## Install and configure Goldstone (as root)
+
+You have two ways to install Goldstone.
+
+### 1. Complete (requires terminal interaction)
+
+The installation script will check the Goldstone prerequisites. If all checks pass, it will then prompt you for additional configuration information. Execute these commands:
+
+```bash
+root# cd /opt/goldstone
+root# . bin/activate
+root# fab full_install
+```
+
+### 2. Partial (requires no terminal interaction)
+
+**N.B.: This section is a work in progress. It is not yet complete.**
```

Comment:
I think nota bene falls under this category:  https://wiki.openstack.org/wiki/Documentation/Conventions#Avoid_obscure_non-English_words_and_abbreviations

The sentences are redundant.